tests/integration/e2e/test_e2e.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import asyncio
+import logging
+
+import pytest
+from pytest_operator.plugin import OpsTest
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.skip_if_deployed
```

Comment:
will not run if `--model=` and `--no-deploy` passed on `tox -e integration-e2e`

Reply:
**praise** I like that we can toggle things using cli !